pkgs/tools/networking/tdns-cli/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
```

usually a newline between attr set argument, and expression
```suggestion
{ stdenv, fetchFromGitHub, rustPlatform }:

```